### PR TITLE
Add Dockerfile to run game review in a Docker container

### PR DESCRIPTION
**Pull request by bluedog8050**
I modified a fork of this project to run in docker on my own server. I believe this small addition would benefit the main project.
### The changes are
- An added Dockerfile which automates building an image that can be run in a container 
- Updates to the readme to document image generation and spinning up a container with the image, and where to add your site key for reCAPTCHA.

The feature simply copies the project into a node 20 docker image and should not require any maintenance independent of the rest of the codebase. Thank you for open sourcing this great little project! Let me know if any changes need to be made or if this is out of scope, this is my first pull request

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,14 +15,33 @@ Enter a game by its PGN or pick a game from your Chess.com / Lichess.org account
 - Run `npm i` to install all of the necessary dependencies.
 - Create a file called `.env` in the root directory of the project.
 - Choose a port for the webserver by adding `PORT=<some port>` to the file.
-- If you want to use a CAPTCHA, add your client secret as `RECAPTCHA_SECRET=<secret>`
+- If you want to use a CAPTCHA:
+    - add your client secret as `RECAPTCHA_SECRET=<secret>` to the .env file
```
Comment (WintrCat) [inline readme.md:19]: Capitalise "add"

File: readme.md [hunk 2]
```diff
@@ -15,14 +15,33 @@ Enter a game by its PGN or pick a game from your Chess.com / Lichess.org account
...
+## Running in Docker
+### Prerequisites
+- Docker installed on the server
+
+### Build a Docker image
+- Download the source code using `git clone` or download as ZIP.
+- Open the root directory of the project in a terminal.
+- Create a file called `.env` in the root directory of the project.
+- If you want to use a CAPTCHA:
+    - add your client secret as `RECAPTCHA_SECRET=<secret>` to the .env file
```
Comment (WintrCat) [inline readme.md:37]: Capitalise "add"

**Review by WintrCat — COMMENTED** [review]
Couple tiny things - then we can probably merge 👍

File: readme.md (hunk 1 above)
Comment (bluedog8050) [reply readme.md]: Done

File: readme.md (hunk 2 above)
Comment (bluedog8050) [reply readme.md]: Done

**Pull request merged**